tests/test_particleContainer.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+import numpy as np
+import amrex
+
+#TODO : Tests struct of arrays
+
+@pytest.fixture()
+def particle_container(std_geometry, distmap, boxarr):
+    pc = amrex.ParticleContainer_1_1_2_1(std_geometry, distmap, boxarr)
+    return pc
+
+def test_n_particles(particle_container):
+    pc = particle_container
+    assert(pc.OK())
+    assert(pc.NStructReal == amrex.ParticleContainer_1_1_2_1.NStructReal == 1)
+    assert(pc.NStructInt == amrex.ParticleContainer_1_1_2_1.NStructInt == 1)
+    assert(pc.NArrayReal == amrex.ParticleContainer_1_1_2_1.NArrayReal == 2)
+    assert(pc.NArrayInt == amrex.ParticleContainer_1_1_2_1.NArrayInt == 1)
+
+
+def test_pc_init():
+    pc = amrex.ParticleContainer_1_1_2_1()
+
+
+    bx = amrex.Box(amrex.IntVect(0, 0, 0), amrex.IntVect(63, 63, 63))
+    rb = amrex.RealBox(0,0,0,1,1,1)
+    coord_int = 1 # RZ
+    periodicity = [0,0,1]
+    gm = amrex.Geometry(bx, rb, coord_int, periodicity)
+
+    ba = amrex.BoxArray(bx)
+    ba.max_size(32)
+    dm = amrex.DistributionMapping(ba)
+
+    pc.Define(gm,dm,ba)
+
+    amrex.ParticleContainer_1_1_2_1(gm,dm,ba)
+
+def test_particle_init(particle_container, std_real_box):
+    pc = particle_container
+
+    myt = amrex.ParticleInitType_1_1_2_1()
+    myt.real_struct_data = [0.5]
+    myt.int_struct_data = [5]
+    myt.real_array_data = [0.5, 0.2]
+    myt.int_array_data = [1]
+
+    Npart = 21
+    iseed = 1
+    pc.InitRandom(Npart,iseed,myt,False,std_real_box)
+    assert(pc.NumberOfParticlesAtLevel(0) == np.sum(pc.NumberOfParticlesInGrid(0))==Npart)
+
+    pc.resizeData()
+
```

Comment:
Assert here that resize worked?

Reply:
I don't know what `resizeData` is supposed to do, actually.  As far as I can tell it is unnecessary but it looked from some example like it needed to be called before getting a particle Tile.  I'll take it out and let something yell at me if it turns out to be necessary

Reply:
@atmyers can you help on this please?